openmdao/core/system.py

```diff
@@ -27,12 +27,18 @@
 from openmdao.utils.array_utils import convert_neg
 from openmdao.utils.record_util import create_local_meta, check_path
 from openmdao.utils.write_outputs import write_outputs
+from openmdao.jacobians.assembled_jacobian import DenseJacobian, CSCJacobian
```

Comment:
Not sure why you don't combine all 3 imports from here into the first import statement.

Reply:
fixed